packages/libp2p/test/dcutr/dcutr.node.ts

```diff
@@ -48,6 +54,9 @@ describe('dcutr', () => {
       services: {
         identify: identifyService(),
         relay: circuitRelayServer()
+      },
+      connectionManager: {
+        minConnections: 0, // disable autodial
```

Comment:
Why is it necessary to disable autodial on the relay?

Reply:
I tried disabling auto-dial on everything. I haven't tested if the test passes with this removed. Will check

Reply:
removed auto-dial disabling on relay